install/kubernetes/helm/subcharts/security/values.yaml

```diff
@@ -5,5 +5,5 @@ enabled: true
 replicaCount: 1
 image: citadel
 selfSigned: true # indicate if self-signed CA is used.
-identityDomain: cluster.local # indicate the domain used in SPIFFE identity URL
+trustDomain: cluster.local # indicate the domain used in SPIFFE identity URL
```

Comment:
/hold 
That would be a backward incompatible / upgrade breaking change.

Reply:
@costin identityDomain was introduced by @elevran on master after 1.0, so within 1.1 it shouldn't be incompatible.

@elevran @myidpt @hklai we should get this one into 1.1, trust domain is the name used in SPIFFE, using a different name in Istio for the same thing would be pretty confusing.

Reply:
@costinm identityDomain is not in V1.0. There shouldn't be compatibility issue if we make this change into V1.1.

Reply:
Thanks for checking - please indicate this clearly in the PR description, we had several breakages and we need to be very careful when reviewing any API change.